plugin/src/main/java/at/helpch/chatchat/util/MentionUtils.java

```diff
@@ -93,7 +94,9 @@ public static MentionReplaceResult replaceMention(
         if (target instanceof ChatUser) {
             final var targetChatUser = (ChatUser) target;
 
-            if (targetChatUser.player().hasPermission(MENTION_CHANNEL_BLOCK_PERMISSION) && !user.player().hasPermission(MENTION_CHANNEL_BLOCK_OVERRIDE_PERMISSION)) {
+            if (!targetChatUser.mentions() &&
+                targetChatUser.player().hasPermission(MENTION_CHANNEL_BLOCK_PERMISSION)
```

Comment:
Do we still want the permission? I feel like it shouldn't be needed anymore. Just give them permission to toggle and that's it. Doesn't make sense to give them permission to toggle and still require another permission for the toggle to work.

Reply:
Currently the toggle command works as a general toggle for both channel & personal mentions. Leaving the permission check allowed for server owners to still limit their ability to toggle to only personal or only channel if they wanted.

I can remove the `.block` permissions entirely and just have the toggle permission/command, but then it gets rid of a bit of control on the server owner side.

Reply:
I see. Might want to think of a better solution. Maybe an extra parameter that tells what mention they want to toggle and with different permission checks per.

Reply:
Yea I considered that as well, but same issue mentioned above that it may be confusing for players to understand the difference between channel and personal mentions, as opposed to just thinking "toggle pings"

Reply:
Yeah I guess that's true. Not sure what best solution would be here. I guess we can keep it like this for now.

Reply:
I pushed a separate ones commit. It isn't actually that confusing with autocomplete, which I forgot about.

I can always roll back to right before this commit if we decide it is too confusing.

Reply:
nice nice.